From 5f8e0876a8466bd9f4c7edf0405c313011b92862 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 2 Mar 2023 13:16:27 +0100
Subject: [PATCH 374/841] feat(stm32mp1-fdts): remove useless /delete-node/
 properties for SSP

NOT_UPSTREAMABLE

With the /omit-if-no-ref/ keyword added to pins node, the explicit
removal of those nodes becomes useless. Remove the corresponding lines
in SSP DT overlays.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 8aa683ae13e6b5b80e5f6562087d7f5633938abd feat(stm32mp1-fdts): remove useless /delete-node/ properties for SSP]
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Change-Id: I25456ba8c78fc71edbad9f416f2ccff50c4b90a2
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/293343
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 fdts/stm32mp13-ssp-bl2.dtsi |  5 -----
 fdts/stm32mp15-ssp-bl2.dtsi | 23 -----------------------
 2 files changed, 28 deletions(-)

diff --git a/fdts/stm32mp13-ssp-bl2.dtsi b/fdts/stm32mp13-ssp-bl2.dtsi
index 8f5fc5b45..0307b9bea 100644
--- a/fdts/stm32mp13-ssp-bl2.dtsi
+++ b/fdts/stm32mp13-ssp-bl2.dtsi
@@ -43,10 +43,5 @@
 #if STM32MP_UART_PROGRAMMER
 		/delete-node/ usbphyc@5a006000;
 #endif
-
-		pinctrl@50002000 {
-			/delete-node/ sdmmc1-b4-0;
-			/delete-node/ sdmmc2-b4-0;
-		};
 	};
 };
diff --git a/fdts/stm32mp15-ssp-bl2.dtsi b/fdts/stm32mp15-ssp-bl2.dtsi
index b8c75c073..8698a9be7 100644
--- a/fdts/stm32mp15-ssp-bl2.dtsi
+++ b/fdts/stm32mp15-ssp-bl2.dtsi
@@ -71,28 +71,5 @@
 		/delete-node/ stgen@5c008000;
 		/delete-node/ i2c@5c009000;
 		/delete-node/ tamp@5c00a000;
-
-		pinctrl@50002000 {
-			/delete-node/ fmc-0;
-			/delete-node/ qspi-clk-0;
-			/delete-node/ qspi-bk1-0;
-			/delete-node/ qspi-bk2-0;
-			/delete-node/ sdmmc1-b4-0;
-			/delete-node/ sdmmc1-dir-0;
-			/delete-node/ sdmmc2-b4-0;
-			/delete-node/ sdmmc2-b4-1;
-			/delete-node/ sdmmc2-d47-0;
-#if STM32MP_USB_PROGRAMMER
-			/delete-node/ uart7-0;
-			/delete-node/ uart7-1;
-			/delete-node/ usart2-0;
-			/delete-node/ usart3-0;
-			/delete-node/ usart3-1;
-#endif
-#if STM32MP_UART_PROGRAMMER
-			/delete-node/ usbotg_hs-0;
-			/delete-node/ usbotg-fs-dp-dm-0;
-#endif
-		};
 	};
 };
-- 
2.47.2

